binary/capabilities.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.substrait;
+
+option java_multiple_files = true;
+option csharp_namespace = "Substrait.Protobuf";
+
+// Defines a set of Capabilities that a system (producer or consumer) supports.
+message Capabilities {
+
+    // List of Substrait versions this system supports
+    repeated string substrait_versions = 1;
+
+    // list of com.google.Any message types this system supports for advanced extensions.
```

Comment:
What does `com.google.Any` have to do with this field?

Reply:
com.google.Any defines [two fields](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto#L154).

One is the type the any is holding (`type_label`) and the other is the binary data for that message type (`value`). This is a list of all `type_label`s that may be used in this plan. It allows a consuming system to understand whether there may be unexpected messages types without having to walk the entire tree.